index.html

```diff
@@ -8,8 +8,10 @@
 
 <div class="sg-c">
 	<div class="sg-component">
-
-		<p class="hero-quote icon icon-aperature"> Gender Equity in Tech Conference, GETconf for short, is a one-day conference to amplify + celebrate the voices of women & gender expansive individuals in tech.</p>
+		  <h2 class="group-hed group-hed-r">Who We Are</h2>
+			<div class="hero-quote icon icon-aperature">
+					<p>Gender Equity in Tech Conference, GETconf for short, is a one-day conference to amplify + celebrate the voices of women & gender expansive individuals in tech.</p>
```

Comment:
I also changed this on Twitter from:

`is a one-day conference`

to

`is a series of events`

I'm not tied to that verbiage, but since we're more than a 1-day event now, we should consider updating this description.

Reply:
Good idea!